gazelle/python/testdata/from_imports/import_nested_var/__init__.py

```diff
@@ -13,8 +13,4 @@
 # limitations under the License.
 
 # baz is a variable in foo/bar/baz.py
-from foo\
-    .bar.\
-        baz import (
-            baz
-        )
+from foo.bar.baz import baz
```

Comment:
This was intentionally formatted oddly to test the parser. Please wrap in `# fmt: off`